crates/csharp/src/lib.rs

```diff
@@ -207,7 +224,7 @@ impl WorldGenerator for CSharp {
         _files: &mut Files,
     ) -> Result<()> {
         let name = &format!("{}-world", resolve.worlds[world].name);
-        let mut gen = self.interface(resolve, name, false);
+        let mut gen = self.interface(resolve, name, false, false);
```

Comment:
could we alias the true/false to something?  It is hard to know what the intent is here

Reply:
good call, in fact the second one was no longer used, and we can use the existing `Direction` for the first one.  Have made that change

Reply:
edit, ok, that flag is needed, have made them both enums to make the code a bit more readable.